src/app/web-app.component.ts

```diff
@@ -146,6 +146,16 @@ export class WebAppComponent implements OnInit {
     if (!localStorage.getItem('mifosXDateFormat')) {
       this.settingsService.setDateFormat('dd MMMM yyyy');
     }
+    if (!localStorage.getItem('mifosXServers')) {
+      this.settingsService.setServers([
+        'https://dev.mifos.io',
+        'https://demo.mifos.io',
+        'https://staging.mifos.io',
+        'https://mobile.mifos.io',
+        'https://demo.fineract.dev',
+        'https://localhost:8443'
+      ]);
+    }
```

Comment:
Here is the list of servers we are offering. Customizers can change this list to their own servers before build time.